include/networkit/community/PLM.hpp

```diff
@@ -33,7 +34,7 @@ class PLM final : public CommunityDetectionAlgorithm {
      * @param[in] recurse use recursive coarsening, see http://journals.aps.org/pre/abstract/10.1103/PhysRevE.89.049902 for some explanations (default: true)
      *
      */
-    PLM(const Graph& G, bool refine=false, double gamma = 1.0, std::string par="balanced", count maxIter=32, bool turbo = true, bool recurse = true);
+    PLM(const Graph& G, bool refine=false, double gamma = 1.0, std::string par="balanced", count maxIter=32, bool turbo = true, bool recurse = true, Partition zeta=Partition(0));
```

Comment:
Adding an initial partition looks like a good idea. However, the parameter list is already quite long and setting the partition this way seems inconvenient. What about adding a `setInitialPartition` method? This method could also be overloaded to also take an rvalue ref, so that we can avoid unnecessary copies.